test/features/user_can_read_robot_statistics_hiring_test.rb

```diff
@@ -0,0 +1,216 @@
+require_relative "../test_helper"
+class UserCanReadRobotStatsHiringTest < FeatureTest
+
+  def test_user_can_see_hiring_stats
```

Comment:
same here. although, this may be a question of what you're trying to test. feature tests should test a user visiting your application. it should not test every individual choice. you should be able to assume that regardless of what they fill in, the output should be relatively the same. 

you should test to see that hiring stats for different robots are working correctly in a unit test.